src/java/org/apache/cassandra/schema/TableMetadata.java

```diff
@@ -461,57 +461,125 @@ public static boolean isSuperColumnMapColumnName(ByteBuffer columnName)
         return !columnName.hasRemaining();
     }
 
+    /**
+     * Method that compares two TableMetadata objects. This is a modified version of validateCompatibility that is used
+     * when checking the compatibility between the schema metadata from an SSTable against the schema metadata from a
+     * CQL table.
+     *
+     * The serialization header of the SSTable does not contain exactly the same information available in the schema of
+     * a CQL table, so the comparison needs to be adapted to the information available.
+     *
```

Comment:
You need to add `<p>` otherwise it will not show the new line.
```suggestion
     * <p>
```

Reply:
Done.